03_python_fastapi_project/main.py

```
@@ -126,6 +141,101 @@ async def delete_product(product_id: int, db: AsyncSession = Depends(get_db)):
     return {"message": "Product deleted successfully"}
 
 
+# Cart endpoints
+@app.get("/cart", response_model=List[CartItemDTO])
+async def get_cart(db: AsyncSession = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)